snuba/clusters/cluster.py

```diff
@@ -241,6 +241,12 @@ def is_single_node(self) -> bool:
         """
         return self.__single_node
 
+    def get_cluster_name(self) -> Optional[str]:
```

Comment:
just `get_name` ? `cluster.get_cluster_name()` is redundant.

Reply:
The reason is it's the name of the users ClickHouse cluster, which can be different to the Snuba cluster concept (yes it's a bit confusing).

Reply:
then `get_clickhouse_name` ?  Or `get_db_cluster_name` ?

Reply:
How about `get_clickhouse_cluster_name()` - a bit long winded but maybe least likely to be confused